technology/block-rotation-spec.adoc

```diff
@@ -0,0 +1,80 @@
+= Block Rotation Specification
+
+* SKALED maintains not an exact amount of blocks set in the config, but at least this amount.
```

Comment:
what is `this amount`?

Reply:
this amount is defined in the very next bullet